eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -18,6 +18,9 @@ parameters:
   type: boolean
   default: false
 
+variables:
```

Comment:
For this case it looks like they are already including our shared client so it shouldn't be needed here.

Reply:
In fact you cannot add it see https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2188656&view=results

Reply:
I've taken it out.

Reply:
@kurtzeborn in order to fix the current cosmos issue you might need to update to use extends in the cosmos-sdk-client.yml. Only question is if you can do a double extends.

Reply:
You would need to attempt to change https://github.com/Azure/azure-sdk-for-python/blob/main/eng/pipelines/templates/stages/cosmos-sdk-client.yml#L21 to extends. However I'm not sure if that will work and even if it does work I'm not sure how that will work when they re-enable their emulator tests which are currently commented out.

Reply:
I think we fixed it now.  Thanks for the help!